views/Data.js

```diff
@@ -32,16 +32,22 @@ let reducer = produce((draft, action) => {
   switch (action.type) {
     case SET: {
       set(draft.value, action.path, action.value)
+      if (action.touched !== false) {
```

Comment:
should we change this to `if (action.touched) {` and not even pass `touched: false` in the dispatch below when it isn't needed?